parser-typechecker/src/Unison/Codebase/Editor/RemoteRepo.hs

```diff
@@ -117,7 +120,7 @@ data ReadGitRemoteNamespace = ReadGitRemoteNamespace
 
 data ReadShareRemoteNamespace = ReadShareRemoteNamespace
   { server :: ShareCodeserver,
-    repo :: Text,
+    repo :: ShareUserHandle,
```

Comment:
question: this name "repo" is baked into json somewhere, right?

Reply:
I don't believe so, AFAIK we prepend it to the `path` parameter on sync requests.